ci/Jenkinsfile-pyp

```diff
@@ -0,0 +1,205 @@
+pipeline {
+    agent any
+
+    parameters {
+        choice(
+            name: 'release_type',
+            choices: ['dev', 'stable'],
+            description: 'Indicate whether a dev or stable release should be built.'
+        )
+        choice(
+            name: 'version_increment',
+            choices: ['patch', 'minor', 'major'],
```

Comment:
@arnaudsjs do we also need an option for no increment, when re-building a previously tagged release (e.g. pypi upload problems, ...)

Reply:
Yes. That would be handy. Just in case the pipeline fails after the versionbump step.